common/test/acceptance/pages/lms/discussion.py

```diff
@@ -354,12 +354,11 @@ def close_open_thread(self):
         with self.thread_page._secondary_action_menu_open(".forum-thread-main-wrapper"):
             self._find_within(".forum-thread-main-wrapper .action-close").first.click()
 
-    @wait_for_js
-    def is_window_on_top(self):
+    def is_focus_changed(self, selector):
         """
-        Check if window's scroll is at top
+        Check if the focus changed
         """
-        return self.browser.execute_script("return $('html, body').offset().top") == 0
+        return self.browser.execute_script("return $('{}').is(':focus')".format(selector))
```

Comment:
I'm confused by this. Does this tell me the focus is set somewhere on the html? Or perhaps on the entire body of the html (which implies that the focus is not set)?